### ISSUE TITLE
Callback when image is loaded

### ISSUE THREAD

**Issue by rimlin** [tjoskar/ng-lazyload-image#176]
Hello, what about implement option to get notify when image was loaded? It will be very useful in many cases.
I suggest this example of usage:

```
@Component({
    selector: 'image',
    template: `
      <div [hidden]="isLoading">
        <img 
          [defaultImage]="defaultImage" 
          [lazyLoad]="image" 
          [offset]="offset"
          (onLoad)="onImageLoad()" />
      </div>
      <div *ngIf="isLoading">
        <loader></loader>
      </div>
    `
})
class ImageComponent {
    defaultImage = 'https://www.placecage.com/1000/1000';
    image = 'https://images.unsplash.com/photo-1443890923422-7819ed4101c0?fm=jpg';
    offset = 100;
    isLoading = true;

    onImageLoad() {
      this.isLoading = false;
    }
}
```

If the idea is good, then I could make pull request, if needed.

**Comment by tjoskar** [comment]
Hi,
I have been thinking about this and I can see some use case. I gladly accept pull requests :). It should be quite easy to implement since the observer gets emitted when the image is loaded: https://github.com/tjoskar/ng-lazyload-image/blob/master/src/lazyload-image.directive.ts#L37

**Comment by rimlin** [comment]
https://github.com/tjoskar/ng-lazyload-image/pull/177

**Comment by elitenick** [comment]
Need this too! Commenting to so I know when it's merged.

**Comment by tjoskar** [comment]
Fixed in #177

**Comment by tjoskar** [comment]
Included in 3.2.0 
Thanks @rimlin
